.github/workflows/update-changelog.yaml

```diff
@@ -23,10 +28,11 @@ jobs:
         uses: peter-evans/create-pull-request@v3
         with:
           commit-message: "docs(CHANGELOG.md): Updating changelog"
-          committer: GitHub <noreply@github.com>
-          author: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
           signoff: true
           branch: chore/update-changelog
           delete-branch: true
           title: "docs(CHANGELOG.md): Updating changelog"
           draft: false
+          token: ${{ steps.github-app-token.outputs.token }}
+          committer: kumahq[bot] <110050114+kumahq[bot]@users.noreply.github.com>
+          author: kumahq[bot] <110050114+kumahq[bot]@users.noreply.github.com>
```

Comment:
One annoying thing about this is that for some reason it adds the `coauthor-by` tag to the commit for some reason, even though committer and author are the same... Not sure how to get rid of that.